Paper-API-Patches/0013-Add-Consume-Arrow-API-to-EntityShootBowEvent.patch

```diff
@@ -0,0 +1,55 @@
+From 2833141390ace01907cfaabaedd9e594207b4c76 Mon Sep 17 00:00:00 2001
+From: Mystiflow <mystiflow@gmail.com>
+Date: Tue, 5 Jun 2018 00:03:09 +0100
+Subject: [PATCH] Add Consume Arrow API to EntityShootBowEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/entity/EntityShootBowEvent.java b/src/main/java/org/bukkit/event/entity/EntityShootBowEvent.java
+index f8c91a13..5ba868f2 100644
+--- a/src/main/java/org/bukkit/event/entity/EntityShootBowEvent.java
++++ b/src/main/java/org/bukkit/event/entity/EntityShootBowEvent.java
+@@ -15,13 +15,15 @@ public class EntityShootBowEvent extends EntityEvent implements Cancellable {
+     private final ItemStack bow;
+     private Entity projectile;
+     private final float force;
++    private boolean consumeArrow;
+     private boolean cancelled;
+ 
+-    public EntityShootBowEvent(final LivingEntity shooter, final ItemStack bow, final Projectile projectile, final float force) {
++    public EntityShootBowEvent(final LivingEntity shooter, final ItemStack bow, final Projectile projectile, final float force, final boolean consumeArrow) {
```

Comment:
Add back the existing constructor as an overload, for compatibility with Bukkit.

Reply:
Thanks. Made the change.